stack-8.8.3.yaml

```diff
@@ -1,39 +1,32 @@
-resolver: nightly-2020-01-21 # last GHC 8.8.1
+resolver: lts-15.4
 packages:
 - .
 - hie-plugin-api
 
 extra-deps:
 # - ./submodules/HaRe
 
-- aeson-1.4.6.0
 - apply-refact-0.7.0.0
-- brittany-0.12.1.1
-- bytestring-trie-0.2.5.0
+- bytestring-trie-0.2.5.0@rev1
 # - cabal-helper-1.0.0.0
 - github: jneira/cabal-helper
   commit: ffb1f57a5ffc6b7ac3c46a9974c4420a6d2bb9b2
 - clock-0.7.2
 - constrained-dynamic-0.1.0.0
-- extra-1.6.21
 - floskell-0.10.2
-- ghc-lib-parser-ex-8.8.5.3
-- haddock-api-2.23.0
-- haddock-library-1.8.0
-- haskell-lsp-0.20.0.0
-- haskell-lsp-types-0.20.0.0
-- lsp-test-0.10.1.0
-- hie-bios-0.4.0
-- hlint-2.2.11
+# To avoid build error with ghc-8.8.2
+# See https://github.com/haskell/haddock/issues/1123
+- git: https://github.com/haskell/haddock.git
+  commit: be8b02c4e3cffe7d45b3dad0a0f071d35a274d65
+  subdirs:
+    - haddock-api
+# - haddock-api-2.23.0
```

Comment:
Looks like `- haddock-api-2.23.1` would also work — it seems to build with ghc 8.8.2 and later. Dunno if using a released version is preferable.

Reply:
Absolutly! Thanks for pointing it out